Source/WebKit/NetworkProcess/haiku/NetworkProcessMainHaiku.cpp

```diff
@@ -39,7 +39,8 @@ void initializeAuxiliaryProcess<NetworkProcess>(AuxiliaryProcessInitializationPa
 
 int NetworkProcessMainUnix(int argc, char** argv)
 {
-    return AuxiliaryProcessMain<NetworkProcess, AuxiliaryProcessMainBase>(argc, argv);
+    //return AuxiliaryProcessMain<NetworkProcess, AuxiliaryProcessMainBase>(argc, argv);
```

Comment:
for now focussing on webprocess so will probably create a template class for both webprocess and netwrokprocess BApplication

Reply:
Can you explain this me again I couldnt understand this 😕

Reply:
I think AuxiliaryProcessMain is like a "main" function, it does not return until the program is done, and when it returns, the program exits.

So if we do nothing here, the program exits immediately, right? And that's not what we want.

Well, let's work a bit on AuxiliaryProcess and get back to this.

Reply:
Oh ok got it so we could Run our Application inside AuxProcessMain so it exists only after its done